### DISCUSSION TITLE
Watch forgotten - Mobil available - how to switch sensor-watch connection to sensor-mobil back

### DISCUSSION THREAD

**Discussion by kdupke** [j-kaltes/Juggluco#238] [Q&A] [answered]
Hi there!

I usually run juggluco in a tandem with my wearOS watch and connect the sensor to the watch (better connection reliability during night, when sometimes the body is between the sensor and the mobile).

Today I get off my home but left my watch at home.
Of course after a while I mentioned that I miss my sensor data.

Now I was far away from my watch and my simple question is:
When juggluco expect data from the watch klon, how to I force juggluco to switch back to directly connect to the sensor?

When I was back home, I thought now everything would be fine. However, it seems that the mobile and the watch did not reconnect. 
Question: Is it right that I had to re-init the watch? What other effects does this have other that it works again?

Thank you for reading so far & best regards

**Comment by j-kaltes** [comment]
You watch and phone need to be in sync before you change "Direct sensor-watch connection".
You can press on Mirror->Sync and Mirror->Reinit as often as you like without negative consequences. Sometimes you also need to turn WIFI or Bluetooth off and on in Android or WearOS settings. It is also possible that the Galaxy Wearable app or Pixel watch app, shows that the watch isn't connected to the phone, then you have to solve that. In rare cases a reboot is necessary.
Is is also possible that accidentally some mirror settings are modified, you can set everything to the defaults, by pressing "Defaults"
That isn't without consequences and can cause data to be resend  from phone to watch, which can be very slow when everything goes over Bluetooth. You have to wait on that.

**Comment by kdupke** [comment]
Thank you - that helps with the second question.

A bit more pressing is the first one, how to fore the phone to take over the connection from the watch, when the watch isn't available.

I could imagine this also may apply the other way round, in case someone left the phone but has the watch 'at hand'.

**Comment by j-kaltes** [comment, marked as answer]
- On the phone, check left menu->Sensor->"Use Bluetooth".
- In left middle menu->Mirror, select the watch connection and _Modify_: uncheck "_Receive from_" and check "_Stream_"
- If the watch comes back, you can in left menu->Mirror, uncheck "_Stream_".
If later the watch has received the values from the phone, you can again use left menu->Watch->Wear OS->Config->"_Direct sensor-watch connection_" to get the right settings. Maybe you have to switch it off and on.


EDIT: you should also switch off left menu->Sensor->"_use Bluetooth_" on the watch, or do that by immediately turning on and off left menu->Watch->Wear OS->Config->"_Direct sensor-watch connection_" on the phone when the watch is connected again with the phone.

**Comment by kdupke** [comment]
Thank you very much!

**Comment by kdupke** [comment]
Sorry I have to run a follow-up on this.
I had a power issue with the watch, switched to the phone for the sensor connection. That worked well.
Now the watch claims 'no glucose from mirror'. 
I tried to re-init, but that does not change any.
Can I re-setup the watch connection without loosing data? If so, how?

**Comment by j-kaltes** [comment]
Because the last glucose values are received on the phone it is very easy. 
Press left menu->Watch->WearOS config->Defaults. 
If that doesn't work, you can also uninstall the WearOS version and remove the WearOS connection in left middle menu->Mirror. Hereafter you install the WearOS version again.
It is also possible that there is nothing wrong with the connection settings, see "Connection watch-phone" in
https://www.juggluco.nl/JugglucoWearOS/intro/index.html
